tests/mocks/dashboard_create_with_filters_with_empty_alias_failure.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
If there's no HTTP requests for these tests can we just remove them?

Reply:
Deleted